solidity-v1/dashboard/src/components/AddToCalendar.jsx

```diff
@@ -0,0 +1,132 @@
+import React from "react"
+import * as moment from "moment"
+import { CompoundDropdown as Dropdown } from "./Dropdown"
+import * as Icons from "./Icons"
+import { ADD_TO_CALENDAR_OPTIONS } from "../constants/constants"
+
+const makeTime = (time) => {
+  return moment(time)
+    .toISOString()
+    .replace(/[-:]|\.\d{3}/g, "")
+}
+
+const makeUrl = (url, event) => {
+  let finalUrl = url + "?"
+  Object.entries(event).forEach(([key, value], index) => {
+    if (!!value) {
+      finalUrl = finalUrl + `${key}=${encodeURIComponent(value)}`
+      if (index !== Object.entries(event).length - 1) {
+        finalUrl = finalUrl + "&"
+      }
+    }
+  })
+  return finalUrl
+}
+
+const makeGoogleCalendarUrl = (event) => {
+  return makeUrl("https://calendar.google.com/calendar/render", {
+    action: "TEMPLATE",
+    dates: `${makeTime(event.startsAt)}/${makeTime(event.endsAt)}`,
+    location: event.location,
+    text: event.name,
+    details: event.details,
+  })
+}
+
+const makeICSCalendarUrl = (event) => {
+  const components = ["BEGIN:VCALENDAR", "VERSION:2.0", "BEGIN:VEVENT"]
+
+  if (typeof document !== "undefined") {
+    components.push(`URL:${document.URL}`)
+  }
+
+  components.push(
+    `DTSTART:${makeTime(event.startsAt)}`,
+    `DTEND:${makeTime(event.endsAt)}`,
+    `SUMMARY:${event.name}`,
+    `DESCRIPTION:${event.details}`,
+    "END:VEVENT",
+    "END:VCALENDAR"
+  )
+
+  return encodeURI(`data:text/calendar;charset=utf8,${components.join("\n")}`)
+}
+
+const options = [
+  {
+    label: "Google Calendar",
+    value: ADD_TO_CALENDAR_OPTIONS.GOOGLE_CALENDER,
+  },
+  {
+    label: "Apple Calendar",
+    value: ADD_TO_CALENDAR_OPTIONS.APPLE_CALENDAR,
+  },
+]
+
+const AddToCalendar = ({
+  name = "New Event",
+  details = "Event details",
+  startsAt = moment().format("YYYY-MM-DD HH:mm:ss"), // date in YYY-MM-DD HH:mm:ss format
```

Comment:
So if `YYYY-MM-DD HH:mm:ss` format is required this component should be responsible for formatting data to a required format. So let's pass  `startsAt` and `endsAt` as a timestamp. What do you think? 🤔

Reply:
Fixed in 59fca295e45bc5b8536c5f641b446ef1213231fe